charts/kong/CHANGELOG.md

```diff
@@ -11,6 +11,14 @@
   for use when the external Service and container listens should differ, such
   as when terminating TLS at a LoadBalancer.
   [#1021](https://github.com/Kong/charts/pull/1021)
+* Added an `ingressController.admissionWebhook.filterSecrets` option. This
+  limits the admission webhook to only the Secrets that KIC has validation
```

Comment:
Could we add some words to tell that webhook will validate secrets with `konghq.com/validate` annotations when it is enabled?